api/src/Modules/registration/Controller/Ticket/BaseTicketInclude.php

```diff
@@ -0,0 +1,73 @@
+<?php declare(strict_types=1);
+/*.
+    require_module 'standard';
+.*/
+
+namespace App\Modules\registration\Controller\Ticket;
+
+use Slim\Container;
+use Slim\Http\Request;
+use Slim\Http\Response;
+
+abstract class BaseTicketInclude extends BaseTicket
+{
+
+
+    public function __construct(Container $container)
+    {
+        parent::__construct($container);
+
+    }
+
+
+    public function processIncludes(Request $request, Response $response, $args, $values, &$data)
+    {
+        if (empty($values)) {
+            return;
+        }
+        if (in_array('ticketType', $values)) {
```

Comment:
This is fine for now, but we probably want a way to DRY this pattern up and move it to BaseController.